tests/acceptance/TestHelpers/CollaborationHelper.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Amrita Shrestha <amrita@jankaritech.com>
+ * @copyright Copyright (c) 2024 Amrita Shrestha amrita@jankaritech.com
+ */
```

Comment:
other code usage text?
```suggestion
 * @copyright Copyright (c) 2024 Amrita Shrestha amrita@jankaritech.com
 ...
 */
```